<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageIssuingCompany</class>
 <widget class="QDialog" name="ManageIssuingCompany">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quản lý công ty phát hành</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../asset/asset.qrc">
    <normaloff>:/asset/images/icon_library.png</normaloff>:/asset/images/icon_library.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Công ty phát hành</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLineEdit" name="input_search_2"/>
         </item>
         <item>
          <widget class="QPushButton" name="btn_search_2">
           <property name="text">
            <string>Tìm kiếm</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="title">
         <string/>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>ID</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="issuing_company_id">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Tên</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="issuing_company_name"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Ngày tạo</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDateEdit" name="issuing_company_created_at"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Cập nhật lần cuối</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDateEdit" name="issuing_company_updated_at"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupaction">
        <property name="title">
         <string>Hành động</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="btn_add_2">
           <property name="text">
            <string>Thêm</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_update_2">
           <property name="text">
            <string>Cập nhật</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_delete_2">
           <property name="text">
            <string>Xóa</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_reset_2">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Danh sách công ty phát hành</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QTableView" name="table_issuing_company"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../asset/asset.qrc"/>
 </resources>
 <connections/>
</ui>
